flink-runtime/src/test/java/org/apache/flink/runtime/scheduler/benchmark/SchedulerBenchmarkBase.java

```diff
@@ -28,9 +28,11 @@ public class SchedulerBenchmarkBase {
     public ScheduledExecutorService scheduledExecutorService;
 
     public void setup() {
-        scheduledExecutorService =
-                Executors.newSingleThreadScheduledExecutor(
-                        new ExecutorThreadFactory("flink-benchmarks"));
+        if (scheduledExecutorService == null) {
```

Comment:
Add document to tell the `scheduledExecutorService` may have been set in subclass.